paddle/operators/conv_shift_op.cu

```diff
@@ -33,9 +33,9 @@ inline int div_up(int x, int y) { return (x + y - 1) / y; }
 // y is fairly small. For large y, it would probably be more efficient
 // to also tile across y.
 template <typename T>
-__global__ void conv_shift_forward(const T *x, const T *y, T *out, int x_width,
-                                   int y_width, int y_half_width,
-                                   int batch_size) {
+__global__ void ConvShiftForward(const T *x, const T *y, T *out, int x_width,
+                                 int y_width, int y_half_width,
+                                 int batch_size) {
```

Comment:
Please reorder parameter.
[Function_Parameter_Ordering](https://google.github.io/styleguide/cppguide.html#Function_Parameter_Ordering)
The following function parameters are the same.